example/scripts/sample_csv_data_loader.py

```diff
@@ -1,23 +1,23 @@
+import sys
 import textwrap
 import uuid
 
-import sys
-from elasticsearch import Elasticsearch
-
 from databuilder.extractor.csv_extractor import CsvExtractor
+from databuilder.extractor.csv_extractor import CSVTableColumnExtractor
 from databuilder.extractor.neo4j_es_last_updated_extractor import Neo4jEsLastUpdatedExtractor
 from databuilder.extractor.neo4j_extractor import Neo4jExtractor
 from databuilder.extractor.neo4j_search_data_extractor import Neo4jSearchDataExtractor
-from databuilder.loader.file_system_elasticsearch_json_loader import FSElasticsearchJSONLoader
-from databuilder.publisher import neo4j_csv_publisher
-from pyhocon import ConfigFactory
-
 from databuilder.job.job import DefaultJob
+from databuilder.loader.file_system_elasticsearch_json_loader import FSElasticsearchJSONLoader
 from databuilder.loader.file_system_neo4j_csv_loader import FsNeo4jCSVLoader
+from databuilder.publisher import neo4j_csv_publisher
 from databuilder.publisher.elasticsearch_publisher import ElasticsearchPublisher
 from databuilder.publisher.neo4j_csv_publisher import Neo4jCsvPublisher
 from databuilder.task.task import DefaultTask
 from databuilder.transformer.base_transformer import NoopTransformer
+from elasticsearch import Elasticsearch
```

Comment:
we should move these two lines(L18, L19) above following the python import rule.

Reply:
if its ok, I will hold off on this as we will be deleting this class per https://github.com/lyft/amundsen/issues/316 anyways